scripts/sysman/ifwiVersion.yml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Intel Firmware Image Version"
+version: "1.8"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Intel Firmware Image Version Name"
```

Comment:
Why is this experimental extension vendor specific?

Reply:
yes, good point, shall follow up on this internally for any alternative solutions